src/composables/permissions.ts

```diff
@@ -0,0 +1,183 @@
+import { METHODS } from '@aeternity/aepp-sdk';
```

Comment:
Maybe we can have a new line after that, same as we have in other files for external and internal dependencies.